websites/D/Discord.style/presence.ts

```diff
@@ -4,12 +4,28 @@ const presence = new Presence({
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
 presence.on("UpdateData", async () => {
+	let presenceData: PresenceData = {
+		largeImageKey: "discordstyle_logo",
```

Comment:
Imgur links are preferred over Discord asset names.
```suggestion
		largeImageKey: "https://i.imgur.com/1wKlRwf.png",
```